tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, **kwargs):
                          default=False,
                          help='restart target instance between tests')
 
+        group.add_option('--locale',
+                         default="undefined",
```

Comment:
So I did put default='en-US' initially, but then it would default to English each time the automation test starts, and this might not be ideal.  Currently it is not mandatory to have language.locale value set in the testvars.json file, and the current default behavior is to go with the device's last language configuration, which is more intuitive

I will clarify the help comment, but it should be "This value overrides the value in testvars.json" since this parameter processing happens after getting the default values from testvars.  I'll change the comment to reflect that.